addons/wowl/static/src/components/notification/notification.scss

```diff
@@ -18,4 +18,30 @@
       }
     }
   }
+
+  .o_toast {
+    max-width: 320px;
+    overflow: hidden;
+    font-size: 1.08333333rem;
+    background-color: rgba(255, 255, 255, 0.7);
+    background-clip: padding-box;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, 0.1);
+    backdrop-filter: blur(10px);
+    border-radius: 0.25rem;
```

Comment:
I was wondering if we want to duplicate bootstrap rules, or simply use it (keep `toast` className, and deal with the opacity trick). Even though we kind of have duplicated rules with your solution, I think it's nice not to rely on bootstrap anymore here. However, you should use variables instead of hardcoding values (see how they define their rules in bootstrap, _toasts.scss). Also, did you check that every rule is necessary? We may want to simplify this a bit.

Reply:
@aab-odoo I tried to keep it as much as it was before, but I'd love to actually get rid of bootstrap as much as possible, so I'll make it with our css code. I believe that if we rewrite most of the JS, we should also refactor most of the css so it's time to make it nice. We should discuss more about this aspect next meeting.